tests/CMakeLists.txt

```diff
@@ -88,11 +90,7 @@ add_executable(kat_sig kat_sig.c)
 target_link_libraries(kat_sig PRIVATE ${API_TEST_DEPS})
 
 add_executable(test_sig test_sig.c)
-if((CMAKE_C_COMPILER_ID MATCHES "Clang") OR (CMAKE_C_COMPILER_ID STREQUAL "GNU"))
-    target_link_libraries(test_sig PRIVATE ${API_TEST_DEPS} Threads::Threads)
-else ()
-    target_link_libraries(test_sig PRIVATE ${API_TEST_DEPS})
-endif()
+target_link_libraries(test_sig PRIVATE ${API_TEST_DEPS})
```

Comment:
Nice simplification: If [FindThreads](http://www.devdoc.net/linux/cmake-3.9.6/module/FindThreads.html) sets `CMAKE_USE_PTHREADS_INIT` this automatically gets activated also for testing and thus doesn't need compiler (version) checks any more.